src/dsp_tools/commands/xmlupload/upload_config.py

```diff
@@ -27,7 +27,7 @@ def _transform_server_url_to_foldername(server: str) -> str:
         r"^api\.": "",
         r":\d{2,5}/?$": "",
         r"/$": "",
-        r"0.0.0.0": "localhost",
+        r"0.0.0.0": "localhost",  # noqa: S104 (hardcoded-bind-all-interfaces)
```

Comment:
Unfortunately, DSP-API and DSP-APP enforce 0.0.0.0 and don't accept localhost. As explained in https://docs.astral.sh/ruff/rules/hardcoded-bind-all-interfaces/, this is subpar, but there's nothing that dsp-tools can do against, so I just ignore it.

Reply:
yes, we've been wanting to fix this for over a year, but it's a SIPI issue 🙄